socorro/external/postgresql/models.py

```diff
@@ -14,11 +14,23 @@
 from sqlalchemy.orm import relationship
 import sqlalchemy.types as types
 
-try:
-    from sqlalchemy.dialects.postgresql import *
-    from sqlalchemy.dialects.postgresql.base import ischema_names
-except ImportError:
-    from sqlalchemy.databases.postgres import *
+from sqlalchemy.dialects.postgresql import (
+    INTERVAL,
+    INTEGER,
+    TEXT,
+    TIMESTAMP,
+    NUMERIC,
+    REAL,
+    DATE,
+    BOOLEAN,
+    UUID,
+    VARCHAR,
+    ARRAY,
+    BIGINT,
+    SMALLINT,
+    CHAR,
+)
+from sqlalchemy.dialects.postgresql.base import ischema_names
```

Comment:
"global solution" meaning something like the fact that do flake8 check in CI of all webapp-django. And that has its own configuration of exceptions.
But we don't do it for all of `socorro` yet. When I worked on this PR I just used `flake8 socorro/external/postgres` and flake8 runs on all files. Perhaps I *could* have added this file to `setup.cfg`. Dang!